pydm/widgets/base.py

```diff
@@ -1186,6 +1302,29 @@ def channels_for_tools(self):
         """
         return self.channels()
 
+    def eventFilter(self, obj, event):
+        """
+        Filters events on this object.
+
+        Params
+        ------
+        object : QObject
+            The object that is being handled.
+        event : QEvent
+            The event that is happening.
+
+        Returns
+        -------
+        bool
+            True to stop the event from being handled further; otherwise
+            return false.
+        """
+        if event.type() == QEvent.Enter:
+            self.setToolTip(self.parseTip(self.pydm_tool_tip))
+            return True
```

Comment:
This filter is causing the tooltips for widgets which had them set prior to this PR to not show up anymore, I think because `self.pydm_tool_tip` is not set for existing ones. Probably just need a small change here or in the setter maybe to keep things backwards compatible.

Reply:
Ok this should be addressed now :)